src/test/java/com/corundumstudio/socketio/TransportTest.java

```diff
@@ -0,0 +1,47 @@
+package com.corundumstudio.socketio;
+
+import com.corundumstudio.socketio.Transport;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.Assert;
+
+public class TransportTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void byNameIllegalArgument() {
+
+    final String value = "????";
+    thrown.expect(IllegalArgumentException.class);
+    Transport.byName(value);
+
```

Comment:
Get rid of the blank lines above & below the method body in the tests.

Reply:
Agreed

Reply:
Done